project-base/storefront/urql/fetcher.ts

```diff
@@ -74,11 +74,24 @@ export const fetcher =
                 body,
             });
 
+            const contentType = result.headers.get('content-type')?.includes('application/json');
```

Comment:
Maybe make the name more descriptive like `isJsonContentType` or smth like that.

Reply:
fixed